engine/src/test/java/com/datatorrent/stram/client/StramClientUtilsTest.java

```diff
@@ -194,6 +201,9 @@ public void getApexDFSRootDirAbsPath() throws IOException
     UserGroupInformation.setLoginUser(testUser);
```

Comment:
Is not this a global change? Any test must be written in such way that it does not affect other tests.

Reply:
Good point. I will rewrite using `PowerMockito.mockStatic` (if possible) to prevent the global change

Reply:
fixed